### ISSUE TITLE
Unused options in GlobalShortcuts::BindShortcuts impl

### ISSUE THREAD

**Issue by dcz-self** [flatpak/xdg-desktop-portal#1311]
### Operating System

n/a

### XDG Desktop Portal version

Git

### Desktop Environment

Other

### Expected Behavior

The spec doesn't have unexplained or unneeded parts.

### Current Behavior

The non-impl side of BindShortcuts takes an option useful for generating the handle. The impl-side call receives a fully-formed handle, so there's no option necessary. Still, there's the "option" argument in the docs, which is completely lacking any description whatsoever.

The actual code for handling this in xdg-desktop-portal silently drops the options on the floor.

The ListShortcuts call doesn't have one, so it seems unnecessary and unnecessarily confusing.

### Steps to Reproduce

1. Go to [org-freedesktop-impl-portal-globalshortcuts-bindshortcuts](https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.impl.portal.GlobalShortcuts.html#org-freedesktop-impl-portal-globalshortcuts-bindshortcuts)
2. Notice "options"
3. Notice lack of description.
...

**Comment by matthiasclasen** [comment]
Yeah, its unused, we should document it as such. Removing it from the xml now would be needlessly disruptive for existing implementations. Could be done as cleanup if we ever rev the interface, though.
